Config/macOS.xcconfig

```diff
@@ -5,3 +5,4 @@ VALID_ARCHS = $(ARCHS) i386
 SKIP_INSTALL = YES
 
 APPCENTER_BUILD_PLATFORM = macOS
+CLANG_ENABLE_MODULE_DEBUGGING = NO
```

Comment:
Please add the comment with explanation why we need it

Reply:
Shouldn't it be set only for Release?

Reply:
Moved for release only